File: python/src/unittest/python/TortillaHarina_tests.py
from ...main.python import TortillaHarina
import unittest

class TortillaHarina_test(unittest.TestCase):
    def setUp(self):
        self.tortillaHarina = TortillaHarina()

    def test_CurrentTemperature(self):
        self.tortillaHarina.setCurrentTemperature(21)
        assertEqual(21,self.tortillaHarina.getCurrentTemperature())

    def test_FalseToast(self):
        self.tortillaHarina.toast(False)
        assertFalse(self.tortillaHarina.isToasted())

    def test_TrueToast(self):
        self.tortillaHarina.toast(True)
        assertTrue(self.tortillaHarina.isToasted())

    def test_Toasting(self):
        assertEqual(10,self.tortillaHarina.getToastTemperature())

File: csharp/unittest-practiceTests/Clases/TortillaHarinaTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using unittestpractice.Clases;

namespace unittest_practiceTests.Clases
{
    [TestClass]
    public class TortillaHarinaTests
    {
        //Asserts
        private readonly TortillaHarina _tortillaHarina = new TortillaHarina();

        [TestMethod]
        public void testCurrentTemperature()
        {
            _tortillaHarina.SetCurrentTemperature(21);
            Assert.AreEqual(21, _tortillaHarina.GetCurrentTemperature());
        }

        [TestMethod]
        public void testFalseToast()
        {
            _tortillaHarina.Toast(false);
            Assert.IsFalse(_tortillaHarina.IsToasted());
        }

        [TestMethod]
        public void testTrueToast()
        {
            _tortillaHarina.Toast(true);
            Assert.IsTrue(_tortillaHarina.IsToasted());
        }

        [TestMethod]
        public void testToasting()
        {
            Assert.AreEqual(40, _tortillaHarina.GetToastTemperature());
        }
    }
}

